Automattic/wp-calypso
# Plans Grid: Advertise Akismet Consistently With Our Marketing Pages

The follow-up to #88271.

![Screenshot](https://github.com/Automattic/wp-calypso/assets/2060580/741d2049-cf98-4b30-8927-f1d976c3fcba)

## Testing Instructions

* Pick a site with plan lower than Creator (Business).
* Visit `/plans/[yoursite].wordpress.com`.
    * Ensure that `Spam protection with Akismet` appears no sooner than under the Creator plan (matching the screenshot above).
* Scroll past the plans grid and click `Compare plans`.
    * `Spam protection with Akismet` should have a checkmark under each of the available plans in the comparison. 

## Pre-merge Checklist
